packages/icons/plugins/svg-dynamic-title-plugin.js

```diff
@@ -0,0 +1,45 @@
+// Plugin to add <title> and <desc> to SVG elements
+const elements = ['svg', 'Svg']
+
+const dynamicTitlePlugin = ({ types: t }) => {
+  const addTitle = {
+    JSXElement(path) {
+      if (
+        !elements.some(element =>
+          path.get('openingElement.name').isJSXIdentifier({ name: element })
+        )
+      ) {
+        return
+      }
+
+      //Remove the Default <title> and <desc> tags if exists
+      path.get('children').forEach(childPath => {
```

Comment:
Curious. Would [`React.Children.forEach`](https://reactjs.org/docs/react-api.html#reactchildrenforeach) do the same logic as `.forEach`?

We can move forward for better a11y, but prefer to adapt React Top-Level API `React.Children.forEach` if same logic. I'll also mention this thought in the upcoming issue.